content/applications/sales/point_of_sale/pricing/electronic_labels.rst

```diff
@@ -150,3 +150,27 @@ successfully sent to Pricer`. If there is any issue, the system displays an erro
 .. warning::
    If a request sent to Pricer fails, Odoo still considers that the product has been updated. In
    that case, we recommend forcing the update of all tags.
+
+
+Discounts
+---------
+
+To display a discount on a Pricer Tag, you have to link a pricelist to the :guilabel:`product variant`
+associated with the tag.
```

Comment:
- I've added a link to the pricelist doc
- I removed the guilabel since it's the concept and not something displayed on screen at that point
- + 100th character
```suggestion
To display a discount on a Pricer Tag, you need to link a :doc:`pricelist <pricelists>` to the
product variant associated with the tag.
```